### ISSUE TITLE
[wg/apa] Accessible Platform Architectures Group Charter

### ISSUE THREAD

**Issue by ruoxiran** [w3c/strategy#498]
New charter proposal, reviewers please take note.

# Charter Review

[Charter](https://w3c.github.io/apa/charter.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https%3A%2F%2Fw3c.github.io%2Fapa%2Fcharter.html)

If applicable:

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2023%2F07%2Fapa-wg-charter&doc2=https%3A%2F%2Fw3c.github.io%2Fapa%2Fcharter.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/apa)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/orgs/w3c/projects/97/views/2).**

## Known or potential areas of concern

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

https://github.com/w3c/apa/issues

## Anything else we should think about as we review? 

Note: proposed chairs should be copied @matatk @JaninaSajka on this issue.

**Comment by plehegar** [comment]
[Advance notice](https://lists.w3.org/Archives/Member/w3c-ac-members/2025JanMar/0074.html)

**Comment by ruoxiran** [comment]
> * in success criteria, PR is about to go, so need to align with the template and also to update additional text with removing PR
> * in coordination, performance HR is no longer available, so need to remove one word for that
> * in participation, CPEC got name changed, and to be updated
> * in decision policy, there is still mention about Director, which should be removed. Also section number of the Process for Vote has changed from 3.4 to 5.2.3.

Thanks for the detailed review. And sorry for failing to align with the new template. All the above issues have been fixed. See the commit at: https://github.com/w3c/apa/commit/971b6a4333c4b59b244b9e26ef477e262899a045

> * would this WG keep in patent policy 2020? or update to the latest May 2025?

I noticed that a new potent policy has been released, so we hope to follow the latest one.

**Comment by jyasskin** [comment]
No concerns from the TAG.

**Comment by simoneonofri** [comment]
No comment or request from SING

**Comment by svgeesus** [comment]
At the end of the proposed charter, it says

> Work in the scope of this group was, before the previous charter, carried out by the [Protocols and Formats Working Group](https://www.w3.org/WAI/PF/charter)

but then, so does[ the current charter](https://www.w3.org/2023/07/apa-wg-charter.html#about) and [the one before that](https://www.w3.org/2021/08/apa-wg-charter.html#about).

Please change "before the previous charter" to something more clear; ideally a link to the actual carter being referred to.

**Comment by ruoxiran** [comment]
Thank you for all comments, please see reply inline:

> I see in the scope that impact of AI is mentioned, but it might have a bit more in the rest of the charter, maybe add a line in 3.4 other deliverables?

Added, please see the commit at: https://github.com/w3c/apa/commit/c78db9816930cda6f7517092e009e5ae91a2ddcf

> Purely editorial: In the first paragraph, s/participatation/participation. 

fixed, please see the commit at: https://github.com/w3c/apa/commit/3fbd7cddb2553538708c782f2d97be0150205351

> In section 4, that sentence from the charter template seems awkward in an accessibility WG: "Each specification should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and recommendations for maximising accessibility in implementations." since the entire docs are about accessiblity. I'd suggest to remove it. 

Removed, please see the commit at: https://github.com/w3c/apa/commit/0b022aba5da1ec657c8c3e9829e1f34fbde77817

> Similar comment in section 5, first sentence, accessibility review being mentioned as part of HR, since they are done by this WG, is probably not needed.

After discussing with APA chairs, we feel that this part of the content should be retained because the documents published by APA will also go through the HR process. Our documents may be developed from a certain Task Force and also require accessibility review by all working groups. At the same time, the accessibility review will sometimes be given to AG, ARIA, WAI IG, etc. for feedback.

**Comment by himorin** [comment]
- [Proposed W3C Charter: Accessible Platform Architectures Working Group (until 2025-07-24/25)](https://lists.w3.org/Archives/Public/public-new-work/2025Jun/0009.html) (public-new-work)
- [VOTE by 2025-07-24/25: Proposed Charter for the Accessible Platform Architectures Working Group](https://lists.w3.org/Archives/Member/w3c-ac-members/2025AprJun/0071.html) (member only)
- [proposed charter](https://www.w3.org/2025/06/apa-wg-charter.html)
